src/components/RichTextEditor/Plugins/index.ts

```diff
@@ -44,6 +45,32 @@ defaultPlugins
         ],
     );
 
+export const columnBreakPlugins = new PluginComposer();
```

Comment:
would it make sense to rename this const to something like "defaultPluginsWithColumns"? with the current name it sounds as if there are only column plugins